gate-core/src/test/groovy/com/netflix/spinnaker/gate/config/AuthConfigTest.groovy

```diff
@@ -95,4 +100,11 @@ class AuthConfigTest extends Specification {
       .collect(Collectors.toList())
     filtered.size() == 1
   }
+
+  private HashMap<Class<?>, Object> getSharedObjects(){
```

Comment:
javadoc please, not only of what this does, but why...and ideally a link to some spring docs that help explain this.

Reply:
The implementation difference comes in constructor of `ExpressionUrlAuthorizationConfigurer` class in version [5.6.x](https://github.com/spring-projects/spring-security/blame/d0491c07e9624964489ba6a5a90ae0547759bc02/config/src/main/java/org/springframework/security/config/annotation/web/configurers/ExpressionUrlAuthorizationConfigurer.java#L108)  and [5.5.x](https://github.com/spring-projects/spring-security/blame/1f481aafff14f324ffe2b43a973d3d5f54ae92d4/config/src/main/java/org/springframework/security/config/annotation/web/configurers/ExpressionUrlAuthorizationConfigurer.java#L105).